src/wallet/wallet2.cpp

```diff
@@ -2263,14 +2264,15 @@ bool wallet2::load_keys(const std::string& keys_file_name, const std::string& pa
   // Decrypt the contents
   r = ::serialization::parse_binary(buf, keys_file_data);
   THROW_WALLET_EXCEPTION_IF(!r, error::wallet_internal_error, "internal error: failed to deserialize \"" + keys_file_name + '\"');
-  crypto::chacha8_key key;
-  crypto::generate_chacha8_key(password, key);
+  crypto::chacha_key key;
+  crypto::generate_chacha_key(password, key);
   std::string account_data;
   account_data.resize(keys_file_data.account_data.size());
-  crypto::chacha8(keys_file_data.account_data.data(), keys_file_data.account_data.size(), key, keys_file_data.iv, &account_data[0]);
+  crypto::chacha20(keys_file_data.account_data.data(), keys_file_data.account_data.size(), key, keys_file_data.iv, &account_data[0]);
```

Comment:
Will changing the number of rounds here break existing wallets?

Reply:
I don't see the context here, but isn't there a chacha8 a couple lines below if error ?

Reply:
sorry, I realized how it worked immediately after hitting send.... you're good.